api/app/tests/integrations/test_pteams.py

```diff
@@ -295,6 +295,282 @@ def test_vuln_ids_are_sorted_correctly(self):
+        # Given
+        # Change status of ticket1
+        json_data = {
+            "topic_status": "completed",
+            "note": "string",
+            "assignees": [self.ticket_response1["user_id"]],
+            "scheduled_at": None,
+        }
+        set_ticket_status(
+            USER1,
+            self.ticket_response1["pteam_id"],
+            self.ticket_response1["ticket_id"],
+            json_data,
+        )
+
+        # When
+        response1 = client.get(
+            f"/pteams/{self.ticket_response1['pteam_id']}/ticket_counts",
+            headers=headers(USER1),
+        )
+        response2 = client.get(
+            f"/pteams/{self.ticket_response1['pteam_id']}/ticket_counts?related_ticket_status=unsolved",
+            headers=headers(USER1),
+        )
+
+        # Then
+        assert response1.status_code == 200
+        assert response1.json()["pteam_id"] == self.ticket_response1["pteam_id"]
+        assert response1.json()["service_id"] is None
+        assert response1.json()["package_id"] is None
+        assert response1.json()["related_ticket_status"] is None
+        assert response1.json()["ssvc_priority_count"] == {
+            "immediate": 2,
+            "out_of_cycle": 0,
+            "scheduled": 0,
+            "defer": 0,
+        }
+
+        assert response2.status_code == 200
+        assert response2.json()["pteam_id"] == self.ticket_response1["pteam_id"]
+        assert response2.json()["service_id"] is None
+        assert response2.json()["package_id"] is None
+        assert response2.json()["related_ticket_status"] == "unsolved"
+        assert response2.json()["ssvc_priority_count"] == {
+            "immediate": 1,
+            "out_of_cycle": 0,
+            "scheduled": 0,
+            "defer": 0,
+        }
+
+    @pytest.mark.parametrize(
+        "exploitation, automatable, service_mission_impact, expected_ssvc_priority_count",
+        [
+            (
+                "active",
+                "yes",
+                "mission_failure",
+                {"immediate": 1, "out_of_cycle": 0, "scheduled": 0, "defer": 0},
+            ),
+            (
+                "public_poc",
+                "yes",
+                "mission_failure",
+                {"immediate": 0, "out_of_cycle": 1, "scheduled": 0, "defer": 0},
+            ),
+            (
+                "none",
+                "no",
+                "mission_failure",
+                {"immediate": 0, "out_of_cycle": 0, "scheduled": 1, "defer": 0},
+            ),
+            (
+                "none",
+                "no",
+                "degraded",
+                {"immediate": 0, "out_of_cycle": 0, "scheduled": 0, "defer": 1},
+            ),
+        ],
+    )
+    def test_it_should_return_correct_ssvc_priority_count_when_ssvc_is_changed(
+        self,
+        exploitation,
+        automatable,
+        expected_ssvc_priority_count,
+        service_mission_impact,
```

Comment:
```suggestion
        service_mission_impact,
        expected_ssvc_priority_count,
```
parametrize側の順序と合わすべき